cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,34 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Results Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <radiogroup class="list-fields" aria-labelledby="show_correctness_label">
```

Comment:
Addressed with 4e44d0e6b7.

I can't get VoiceOver to read me out the number of elements in the group at all, no matter what I do.. what command do you use to get it to say this?